prime-router/metadata/schemas/Experity/GUC-LA-covid-19.schema

```diff
@@ -0,0 +1,45 @@
+# Schema for Greenlight Urgent Care
+# This schema is to be used to parse the standard lab report from experity
+# Created by Herman Cordero
+# Last Update: 07/27/2021
+# 
+name: guc-la-covid-19
+topic: covid-19
+description: Experity CSV lab report schema, Greenlight Urgent Care, LA
+extends: experity/experity-covid-19
+
+elements:
+
+    - name: sender_id
+      default: guc-la
+
+    - name: processing_mode_code
+      default: P
```

Comment:
```suggestion
      default: T
```

Same thing, run them in Training mode until LA is ok with their submissions, then we'll do a PR to switch them to 'P'.  We realize this is an extra step, but its important.   Thanks

Reply:
What is entailed in this new process?  The testing of Greenlight and Compass has already been worked out with LA and AL.  Both LA and AL Depts of Health are waiting on us to send production data for validation.  They are stopping all results on their side; performing the validation; and then will email me when they are approved.